packages/bpk-docs/src/pages/ChipsPage/ChipsPage.js

```diff
@@ -44,22 +46,24 @@ class DefaultChipContainer extends Component<{}, State> {
   }
 
   addRemovableChip = () => {
-    this.setState({
+    this.setState(() => ({
       chipNames: [
         ...this.state.chipNames,
         `Example Chip ${this.state.nextChipId}`,
       ],
       nextChipId: this.state.nextChipId + 1,
-    });
+    }));
   };
 
   removeChip = chipName => {
     const newChipNames = this.state.chipNames.slice(0);
     const indexToRemove = newChipNames.indexOf(chipName);
+
     newChipNames.splice(indexToRemove, 1);
-    this.setState({
+
+    this.setState(() => ({
```

Comment:
Based on Shaun's comment above this could be refactored to use the `prevState` value in the callback too